### PR TITLE
Remove generated API docs from the README

### PR DESCRIPTION
**Pull request by imaustink**
This pull request was created with [Landscaper](https://github.com/bitovi/landscaper).
The following code mods were used to create this PR:

1. **https://gist.github.com/imaustink/f22e65791d3b874680b6c77550bebd32**,1. **https://gist.github.com/imaustink/dc8c702f9940cc3346cc41338e890838**,1. **https://gist.github.com/imaustink/72693934627c9fcc24ffbbb6b5163929**

Please review this PR carefully as Landscaper does not guarantee any code mod's quality.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,321 +1,31 @@
...
-- `0.6.8` - Bugfix: Don't send data with DELETE requests. Thanks @kylegifford!
-- `0.6.7` - Feature: Allow passing of id into `get` as an object literal `{_id: 1}`, in addition to string and number ids.  Thanks @obaidott!
-- `0.6.6` - Bugfix: disable JSON stringify for params, prevent params from being double stringified. Thanks @obaidott!
-- `0.6.0` - it's now possible to turn off socket.io by passing `socketio = false` to the options. This has been renamed as of version `1.0.0`.
-- `0.5.0` - The default cookie name is now `feathers-jwt` to match the Feathers default.
-- `0.4.0`
-  - The `rest` methods now use a Promise instead of a Deferred.
-  - Error responses are now converted to [feathers-errors](http://docs.feathersjs.com/middleware/error-handling.html).
-- `0.2.0` - Added socket.io auth to the `authenticate` method to get authenticated real-time events.
-- `0.1.0` - Initial version
```
Comment (chasenlehara) [inline readme.md]: We should make sure all of these are in GitHub’s release notes.

File: readme.md [hunk 2]
```diff
@@ -1,321 +1,31 @@
...
-  realtime,
-  dataCallbacks
-], {
-  // Pass the feathers client as the `feathersClient` property.
-  feathersClient: feathersClient,
-  idProp: 'exp',
-  // Pass the Session Map
-  Map: Session,
-  name: 'session'
-});
```
Comment (chasenlehara) [inline readme.md]: We should make sure all of these docs & examples are in the docs rendered on canjs.com

Comment (imaustink) [inline readme.md]: They seem to all be there.

File: readme.md (hunk 1 above)
Comment (imaustink) [inline readme.md]: I added the ones that were missing.

**Pull request merged**